src/components/CourseSection/CourseSection.jsx

```diff
@@ -48,7 +48,7 @@ class CourseSection extends React.Component {
             isOpen
           >
             {enrollments.map((courseData) => {
-              const { status } = courseData;
+              const { course_run_status: status } = courseData;
```

Comment:
why is this `const { course_run_status: status }` instead of `const { course_run_status }`?

if we're not using both `status` and `course_run_status` below, does it make sense to drop the `status` here and to keep this as `const { course_run_status }` and then below change the switch to use `course_run_status`?

Reply:
Snake case in JS is typically against convention, but in Python it's the norm. This is renaming that field from the API data to be camelCase. That being said, though, `status` is probably not the best name; I will rename to `courseRunStatus`.